src/SqlVirtualMachine/examples/New-AzSqlVM.md

```diff
@@ -20,3 +21,146 @@ Location	Name		ResourceGroupName
+-AutoBackupSettingFullBackupWindowHour 2 -AutoBackupSettingStorageAccessKey 'keyvalue' -AutoBackupSettingStorageAccountUrl ` 
+'https://storagename1.blob.core.windows.net/' -AutoBackupSettingRetentionPeriod 10 -AutoBackupSettingLogBackupFrequency 60 ` 
+-AutoBackupSettingStorageContainerName 'storagecontainer1'
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine and configure auto backup settings.
+
+### Example 7
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -AutoPatchingSettingDayOfWeek Thursday ` 
+-AutoPatchingSettingMaintenanceWindowDuration 120 -AutoPatchingSettingMaintenanceWindowStartingHour 3 -AutoPatchingSettingEnable
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine and configure auto patching settings.
+
+### Example 8
+```powershell
+$password = "P@ssw0rd"
+$encryptionpassword = $password | ConvertTo-SecureString -AsPlainText -Force 
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'centralus' -SqlConnectivityUpdateSettingConnectivityType private ` 
+-SqlConnectivityUpdateSettingPort 1433 -SqlConnectivityUpdateSettingSqlAuthUpdateUserName sa_user ` 
+-SqlConnectivityUpdateSettingSqlAuthUpdatePassword $encryptionpassword
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine and configure SQL connectivity settings.
+
+### Example 9
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -AssessmentSettingEnable
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine and configure assessment settings.
+
+### Example 10
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -KeyVaultCredentialSettingAzureKeyVaultUrl 'keyvault url' `
+-KeyVaultCredentialSettingCredentialName 'cred1' `
+-KeyVaultCredentialSettingEnable `
+-KeyVaultCredentialSettingServicePrincipalName 'clientid' `
+-KeyVaultCredentialSettingServicePrincipalSecret 'sp secret value'
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine and configure key vault settings.
+
+### Example 11
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -SqlManagementType 'Lightweight'
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine with Lightweight mode.
```

Comment:
We are no longer exposing SQLManagementType so we should remove this example. Please check with Amitabh/PM